packages/@aws-cdk/aws-iam/lib/policy.ts

```diff
@@ -56,6 +56,22 @@ export interface PolicyProps {
    * @default - No statements.
    */
   readonly statements?: PolicyStatement[];
+
+  /**
+   * Whether an `AWS::IAM::Policy` must be created
+   *
+   * Unless set to `true`, this `Policy` construct will not materialize to an
+   * `AWS::IAM::Policy` CloudFormation resource in case it would have no effect
+   * (for example, if it remains unattached to an IAM identity or if it has no
+   * statements). This is generally desired behavior, since it prevents
+   * creating invalid--and hence undeployable--CloudFormation templates.
+   *
+   * In cases where you know the policy must be created and it is actually
+   * an error if no statements have been added to it, you can se this to `true`.
+   *
+   * @default false
+   */
+  readonly mustCreate?: boolean;
```

Comment:
I still don't love this name... I proposed changing it to `lazy?: boolean` in the previous PR, but I'm also open to other suggestions 🙂

Reply:
forceResource?

mustExist?

Reply:
I feel "lazy" is usually used to communicate something else.

Reply:
Maybe just `force`?

Reply:
I was considering `force`, slightly on the fence but I also don't have a better alternative. I can do `force`.

Reply:
forceReify? forceRender?